node/src/AudioLevelObserver.ts

```diff
@@ -84,22 +87,27 @@ export class AudioLevelObserver extends RtpObserver<AudioLevelObserverEvents>
 
 	private handleWorkerNotifications(): void
 	{
-		this.channel.on(this.internal.rtpObserverId, (event: string, data?: any) =>
+		this.channel.on(this.internal.rtpObserverId, (event: Event, data?: Notification) =>
 		{
 			switch (event)
 			{
-				case 'volumes':
+				case Event.AUDIOLEVELOBSERVER_VOLUMES:
 				{
+					const notification = new FbsAudioLevelObserver.VolumesNotification();
+
+					data!.body(notification);
+
 					// Get the corresponding Producer instance and remove entries with
 					// no Producer (it may have been closed in the meanwhile).
-					const volumes: AudioLevelObserverVolume[] = data
-						.map(({ producerId, volume }: { producerId: string; volume: number }) => (
-							{
-								producer : this.getProducerById(producerId),
-								volume
-							}
-						))
-						.filter(({ producer }: { producer: Producer }) => producer);
+					const volumes: AudioLevelObserverVolume[] =
+						utils.parseVector(notification, 'volumes', parseVolume)
```

Comment:
It would be great if this `utils.parseVector()` receives as template the typed values that it will return. This is, something like this (not accurate):

```ts
utils.parseVector<Array<{string, number}>>(notifications, 'volumes', parseVolume)
```